docs/contacts/contacts_overview.rst

```diff
@@ -0,0 +1,84 @@
+.. vale off
+
+Contacts
+#########
+
+.. vale on
+
+Contacts are the central factor of a marketing automation platform. 
+These are all the individuals who have visited your websites or interacted with you in some way.
+
+.. vale off
+
+Contact Types
```

Comment:
```suggestion
Contact types
```

You do not need to turn Vale off here. It would have correctly flagged up the camel case error. Only turn it off when you need to (eg if you have to refer to a feature of Mautic and therefore it needs to be capitalised.